diff --git a/node_modules/@ipld/dag-cbor/src/index.js b/node_modules/@ipld/dag-cbor/src/index.js
index 96abf42..e521937 100644
--- a/node_modules/@ipld/dag-cbor/src/index.js
+++ b/node_modules/@ipld/dag-cbor/src/index.js
@@ -1,5 +1,7 @@
 import * as cborg from 'cborg'
 import { CID } from 'multiformats/cid'
+import { bigIntEncoder, bigIntDecoder } from 'cborg/taglib';
+
 
 // https://github.com/ipfs/go-ipfs/issues/3570#issuecomment-273931692
 const CID_CBOR_TAG = 42
@@ -19,7 +21,7 @@ const CID_CBOR_TAG = 42
  * @param {ByteView<T> | ArrayBufferView<T>} buf
  * @returns {ByteView<T>}
  */
-export function toByteView (buf) {
+export function toByteView(buf) {
   if (buf instanceof ArrayBuffer) {
     return new Uint8Array(buf, 0, buf.byteLength)
   }
@@ -35,7 +37,7 @@ export function toByteView (buf) {
  * @param {any} obj
  * @returns {cborg.Token[]|null}
  */
-function cidEncoder (obj) {
+function cidEncoder(obj) {
   if (obj.asCID !== obj && obj['/'] !== obj.bytes) {
     return null // any other kind of object
   }
@@ -60,7 +62,7 @@ function cidEncoder (obj) {
  *
  * @returns {null}
  */
-function undefinedEncoder () {
+function undefinedEncoder() {
   throw new Error('`undefined` is not supported by the IPLD Data Model and cannot be encoded')
 }
 
@@ -72,7 +74,7 @@ function undefinedEncoder () {
  * @param {number} num
  * @returns {null}
  */
-function numberEncoder (num) {
+function numberEncoder(num) {
   if (Number.isNaN(num)) {
     throw new Error('`NaN` is not supported by the IPLD Data Model and cannot be encoded')
   }
@@ -86,7 +88,7 @@ function numberEncoder (num) {
  * @param {Map<any, any>} map
  * @returns {null}
  */
-function mapEncoder (map) {
+function mapEncoder(map) {
   for (const key of map.keys()) {
     if (typeof key !== 'string' || key.length === 0) {
       throw new Error('Non-string Map keys are not supported by the IPLD Data Model and cannot be encoded')
@@ -101,7 +103,8 @@ const _encodeOptions = {
     Map: mapEncoder,
     Object: cidEncoder,
     undefined: undefinedEncoder,
-    number: numberEncoder
+    number: numberEncoder,
+    bigint: bigIntEncoder
   }
 }
 
@@ -116,7 +119,7 @@ export const encodeOptions = {
  * @param {Uint8Array} bytes
  * @returns {CID}
  */
-function cidDecoder (bytes) {
+function cidDecoder(bytes) {
   if (bytes[0] !== 0) {
     throw new Error('Invalid CID for CBOR tag 42; expected leading 0x00')
   }
@@ -137,6 +140,7 @@ const _decodeOptions = {
   tags: []
 }
 _decodeOptions.tags[CID_CBOR_TAG] = cidDecoder
+_decodeOptions.tags[2] = bigIntDecoder
 
 export const decodeOptions = {
   ..._decodeOptions,
